packages/react-native-reanimated/src/mutables.ts

```diff
@@ -10,6 +10,25 @@ const SHOULD_BE_USE_WEB = shouldBeUseWeb();
 
 type Listener<Value> = (newValue: Value) => void;
 
+/**
+ * Hides the internal `_value` property of a mutable. It won't be visible to:
+ * - `Object.keys`,
+ * - `const prop in obj`,
+ * - etc.
+ *
+ * This way when the user accidentally sends the SharedValue to React, he won't get an obscure
+ * error message.
+ *
+ * We hide for both _React runtime_ and _Worklet runtime_ mutables for uniformity of behavior.
+ */
+function hideInternalValueProp(mutable: Mutable) {
+  'worklet';
+  Object.defineProperty(mutable, '_value', {
+    configurable: false,
```

Comment:
```suggestion
    writable: true,
    configurable: false, // the user is not allowed to change properties of sharedValue._value, the user must reassign it.
```

Reply:
WDYT about adding some description and explicitly marking this as writable?

Reply:
Relevant comment is already present in the function's TSDoc and in the other comments in the file.

Writable will be true by default since we are modifying an already created 'standard' object, I don't think re-setting it here has some real value.